test/fixtures/fixtures.js

```diff
@@ -42,46 +42,52 @@ const tableOutput = Object.setPrototypeOf(Object.assign([
 		'33.3% (3/9)',
 		'11.1% (1/9)',
 		'55.6% (5/9)',
+		'0% (0/9)',
```

Comment:
let's set this to true in one of the repos in the mock data, so the percentage won't be zero?